code/modules/uplink/uplink_items.dm

```diff
@@ -1332,21 +1332,21 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 
 /datum/uplink_item/suits/padding
 	name = "Soft Padding"
-	desc = "Padding to add to a jumpsuit to help against melee and bullets."
+	desc = "Inconspicious padding worn underneath jumpsuits to protect the user against melee and bullets."
 	item = /obj/item/clothing/accessory/padding
 	cost = 2
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
 /datum/uplink_item/suits/kevlar
 	name = "Kevlar sheets"
-	desc = "Kevlar sheets to add to jumpsuit to help against bullets and melee."
+	desc = "Inconspicious kevlar sheets worn underneath jumpsuits to protect the user against bullets and melee."
 	item = /obj/item/clothing/accessory/kevlar
 	cost = 2
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 
 /datum/uplink_item/suits/plastic
 	name = "Plastic sheet"
-	desc = "Plastic body sheet to add to a jumpsuit to help against laser and energy harm."
+	desc = "Inconspicious  plastic body sheet worn underneath jumpsuits to protect the user against laser and energy beams."
```

Comment:
This... Defeats the reason to have the plastic sheet...

Reply:
ok fine I'll recall it to energy dampening. whatever soon.

Reply:
The way plastic protects you against laser is that when it burns the top layer of the plastic the thick smoke absorbs a lot of energy.

Reply:
is it explicitly ablative?